Dockerfile.template.erb

```diff
@@ -12,7 +12,7 @@ FROM debian:stretch-slim
 LABEL maintainer "TAGOMORI Satoshi <tagomoris@gmail.com>"
 LABEL Description="Fluentd docker image" Vendor="Fluent Organization" Version="1.1"
 
-ENV DUMB_INIT_VERSION=1.2.0
+ENV DUMB_INIT_VERSION=1.2.1
```

Comment:
From the test, alpine linux 3.7's dumb-init package is 1.2.0.
Need `if` condition like below:

```
<% if is_alpine %>
ENV DUMB_INIT_VERSION=1.2.0
<% else %>
ENV DUMB_INIT_VERSION=1.2.1
<% end %>
```

Reply:
@repeatedly - done , add these changes ,thanks for the review